board/fischertechnik/TXT/rootfs/opt/ftc/apps/system/display/display.py

```diff
@@ -49,7 +52,7 @@ def on_calibrate_touchscreen(self):
         self.mainWindow = TouchBaseWidget()
         self.mainWindow.show()
         old_window.close()
-        subprocess.run(["sudo", "TSLIB_TSDEVICE=/dev/input/event0", "/usr/bin/ts_calibrate"])
+        subprocess.run(["calibrate-touchscreen calibrate"])
```

Comment:
As disusse above, use
```suggestion
subprocess.run(["/usr/bin/sudo /usr/sbin/calibrate-touchscreen calibrate"])
```
here.
 
And note the full path for both commands - this prevents running the wrong command because of a changed PATH environment entry.